### PR TITLE
ported to topkg

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: pkg/pkg.ml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ocaml
+#use "topfind"
+#require "topkg"
+open Topkg
+
+let () =
+  Pkg.describe "io-page" @@ fun c ->
```
Comment (dsheets) [inline pkg/pkg.ml]: s/io-page/uri/?

Comment (fgimenez) [inline pkg/pkg.ml]: oops sure! :)

**Comment by fgimenez** [comment]
hey @dsheets thanks for your review! 

The `lib/uri_services.ml` and `lib/uri_services_full.ml` files were not added to the repo, I had them locally (presumably generated by my first execution with the oasis version) and all worked fine here. I've added them and all seems to be working now, let me know if anything else is needed.

Cheers :)

**Comment by dsheets** [comment]
I don't think the services files should be in the repo or removed from `.gitignore` -- they are generated with `etc/gen_services.ml` which needs to be used in the build system.

Thanks for your contribution!

**Comment by fgimenez** [comment]
@dsheets aha, I hadn't noticed how these files were handled, I've added a hook in the build that calls the gen.sh script and all seems to be ok, plz have a look.

Thanks!

**Comment by dsheets** [comment]
Great! Thanks!

**Pull request merged**